frontend/src/component/ListItem.js

```diff
@@ -1,111 +1,229 @@
-                            </p>
-                        </div>
-                        <div>
-                            <p>
-                                {currentSurfData?.swellPeriod.sg} s
-                            </p>
+        <Wrapper >
+            <div className="content" >
+                <Link className="link" to={{
+                pathname: "/" + spot.id,
+                state: {favourite: isFavourite}
+            }}>
+                    <div >
+                        <div className="title"  >
+                            <div className="title" >
+                                {spot.spotLocationDetails.name}
+                            </div>
                         </div>
-                        <div>
-                            <p>
-                                {currentSurfData?.windSpeed.sg} km/h
+                        <div className="subtitle" >
+                            <p className="text">
+                                {spot.spotLocationDetails.continent}, {spot.spotLocationDetails.country}, {spot.spotLocationDetails.region}
                             </p>
                         </div>
-                    </Box>
-                </Box>
-            </CardContent>
-        </Card>
+                        <div className="databox" >
+                            <div className="airTemp">
+                                <div>
+                                    <FaTemperatureLow size={20}/>
+                                </div>
+                                <p>
+                                    {currentSurfData?.airTemperature.sg} °C
+                                </p>
+                            </div>
+                            <div className="waterTemp">
+                                <div >
+                                    <IoWaterOutline size={20}/>
+                                </div>
+                                <p>
+                                    {currentSurfData?.waterTemperature.sg} °C
+                                </p>
+                            </div>
+                            <div className="height">
+                                <div >
+                                    <BiWater size={20}/>
+
+                                </div>
+                                <p>
+                                    {currentSurfData?.swellHeight.sg} m
+                                </p>
+                            </div>
+                            <div className="period">
+                                <div >
+                                    <MdTimer size={20}/>
+
+                                </div>
+                                <p>
+                                    {currentSurfData?.swellPeriod.sg} s
+                                </p>
+                            </div>
+                            <div className="wind">
+                                <div >
+                                    <RiWindyFill size={20}/>
```

Comment:
das steht doch alles weiter oben schonmal oder?

Reply:
Im Header der Details stehen ein paar mehr werte als in der Listenansicht. Daher doppelt.